Sources/PointFree/Account/Account.swift

```diff
@@ -318,6 +318,77 @@ private let subscriptionOverview = View<AccountData> { data -> [Node] in
   }
 }
 
+private let privateRssFeed = View<AccountData> { data -> [Node] in
+  guard Current.features.hasAccess(to: .podcastRss, for: data.currentUser) else { return [] }
```

Comment:
this view is what shows the RSS link to the user in their account page. it looks like this:

<img width="668" alt="image" src="https://user-images.githubusercontent.com/135203/46319509-481f7980-c5a8-11e8-8424-212e5a91375e.png">

let me know if you have ideas for better copy.